lineman/app/services/mention_service.coffee

```diff
@@ -0,0 +1,7 @@
+angular.module('loomioApp').factory 'MentionService', ->
```

Comment:
Yep, because we're only preparing the clone for editing, no touching actual
records.

On Wed, Oct 28, 2015 at 8:58 PM, Robert Guthrie <notifications@github.com>
wrote:

> In lineman/app/services/mention_service.coffee
> <https://github.com/loomio/loomio/pull/2645#discussion_r43302040>:
>
> > @@ -0,0 +1,7 @@
> > +angular.module('loomioApp').factory 'MentionService', ->
>
> Yep, I think I know what's going on. I was talking about the serializer
> modifying the body before the client gets it.
>
> If we serialized the mentioned usernames into a comment property then this
> might be able to be done client side without modifying the text for edit..
> IE: you could just have renderMentions and edit would not need anything.
>
> Aside:
> Will this work ok with the change detection, "are you sure you want to
> cancel?" stuff?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2645/files#r43302040>.
>